### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: src/gdal/mapper-osmconf.ini [hunk 1]
```diff
@@ -29,7 +29,7 @@ osm_changeset=no
 # keys to report as OGR fields
 attributes=access,aerialway,aeroway,amenity,area,barrier,bridge,building,cycleway,foot,geological,highway,historic,intermittent,landuse,leaf_type,leisure,location,man_made,military,name,natural,pipeline,power,railway,ruins,sport,surface,shop,smoothness,ref,tourism,tracktype,tunnel,waterway,wood:age,wood:density
 # keys that, alone, are not significant enough to report a node as a OGR point
-unsignificant=created_by,converted_by,source,time,ele,attribution
+insignificant=created_by,converted_by,source,time,ele,attribution
```
Comment (dg0yt) [inline src/gdal/mapper-osmconf.ini:32]: I'm sorry, but this GDAL syntax:
https://github.com/OSGeo/gdal/blob/93395860dd62e3850ed31b4c77c51a93fe2919c8/gdal/ogr/ogrsf_frmts/osm/ogrosmdatasource.cpp#L3613-L3614

**Review by dg0yt — CHANGES_REQUESTED** [review]
Thank you very much.

Can you drop the osmconf-related commit, or shall I cherry-pick?

File: src/gdal/mapper-osmconf.ini (hunk 1 above)
Comment (dg0yt) [reply src/gdal/mapper-osmconf.ini:32]: FTR, osmconf.ini is ignored in our configuration, for this particular reason.
https://github.com/OpenOrienteering/mapper/blob/master/codespell.sh

**Comment by luzpaz** [comment]
commit reverted, PR squashed and rebased

**Review by dg0yt — APPROVED** [review]
Thanks.

**Pull request merged**